CHANGELOG.md

```diff
@@ -1,14 +1,18 @@
-# dbt_fivetran_utils v0.2.8
-
-## 🚨 Breaking Changes
-- n/a
+# dbt_fivetran_utils v0.2.9
 
 ## Bug Fixes
-- n/a
+- Added a specific Snowflake macro designation for the `json_extract_path` macro. ([#50](https://github.com/fivetran/dbt_fivetran_utils/pull/50))
+    - This Snowflake version of the macro includes a `try_parse_json` function within the `json_extract_path` function. This allows for the macro to succeed if not all fields are a json object that are being passed through. If a field is not a json object, then a `null` record is generated. 
+- Updated the Redshift macro designation for the `json_extract_path` macro. ([#50](https://github.com/fivetran/dbt_fivetran_utils/pull/50))
+    - Similar to the above, Redshift cannot parse the field if every record is not a json object. This update converts a non-json field to `null` so the function does not fail.
+
+## Under the Hood
+- Included a `union_schema_variable` and a `union_database_variable` which will allow the `source_relation` and `union_data` macros to be used with varying variable names. ([#49](https://github.com/fivetran/dbt_fivetran_utils/pull/49))
+    - This allows for dbt projects that are utilizing more than one dbt package with the union source feature to have different names and not see duplicate variable name errors.
```

Comment:
I feel like this is just a bit clunky and would instead say  "`different variable names and not see duplicate errors.`"